src/main/java/org/apache/commons/lang3/text/translate/LookupTranslator.java

```diff
@@ -46,15 +53,19 @@
      *
      * @param lookup CharSequence[][] table of size [*][2]
      */
-    public LookupTranslator(final CharSequence[]... lookup) {
+    @SuppressWarnings({"enhancedfor.type.incompatible","argument.type.incompatible"}) /*
+    #1 CharSequence is of the type [*][2] (according to the documentation), hence Charsequence has length 2
+    #2 Minimum 1 row ensures seq[0].charAt(0) to be valid
```

Comment:
Having at least one row doesn't mean that there is a `charAt` position 0.

Reply:
It has 2 columns for sure, hence minimum 1 row should ensure the existence of charAt position 0